src/common/ExportSettingsModal/ExportSettingsModalContainer.js

```diff
@@ -5,23 +5,30 @@ import moment from 'moment';
 
 import { stripesConnect } from '@folio/stripes/core';
 import { exportToCsv } from '@folio/stripes/components';
+import { useCustomFields } from '@folio/stripes/smart-components';
 import {
+  CUSTOM_FIELDS_ORDERS_BACKEND_NAME,
   useShowCallout,
 } from '@folio/stripes-acq-components';
 
-import { getExportData, exportManifest } from './utils';
 import ExportSettingsModal from './ExportSettingsModal';
 import {
-  EXPORT_LINE_FIELDS,
-  EXPORT_ORDER_FIELDS,
-} from './constants';
+  exportManifest,
+  getExportData,
+  getExportLineFields,
+  getExportOrderFields,
+} from './utils';
 
 const ExportSettingsModalContainer = ({
   onCancel,
   mutator,
   fetchOrdersAndLines,
 }) => {
   const [isExporting, setIsExporting] = useState(false);
+  const [customFieldsPO, isLoadingPO] = useCustomFields(CUSTOM_FIELDS_ORDERS_BACKEND_NAME, 'purchase_order');
+  const [customFieldsPOL, isLoadingPOL] = useCustomFields(CUSTOM_FIELDS_ORDERS_BACKEND_NAME, 'po_line');
+  const customFields = [...customFieldsPO || [], ...customFieldsPOL || []];
```

Comment:
It probably makes sense to wrap it in `useMemo` since `customFields` is a dependency for the `onExportCSV`, and as a result, `onExportCSV` will be recreated on each render.

Reply:
Done.